flytekit/configuration/default_images.py

```diff
@@ -35,13 +35,16 @@ def default_image(cls) -> str:
             if default_image is not None:
                 return default_image
 
-        default_image_str = os.environ.get("FLYTE_INTERNAL_IMAGE", cls.find_image_for())
-        return default_image_str
+        return cls.find_image_for()
 
     @classmethod
     def find_image_for(
         cls, python_version: typing.Optional[PythonVersion] = None, flytekit_version: typing.Optional[str] = None
     ) -> str:
+        default_image_str = os.getenv("FLYTE_INTERNAL_IMAGE")
```

Comment:
`FLYTE_INTERNAL_IMAGE` is not really documented. What is it used for?

Reply:
It's very useful when I need to override the default image for the task or the base image in the ImageSpec.

Reply:
can we move `FLYTE_INTERNAL_IMAGE` to a constant and leave this comment there?